ports/stm32/boards/Passport/modules/tasks/search_for_address_task.py

```diff
@@ -54,6 +55,7 @@ async def search_for_address_task(
                     if curr_address == address:
                         await on_done(curr_idx, addr_path, None)
                         return
+                    await sleep_ms(1)
```

Comment:
We probably need a similar sleep above on line 42 for the multisig case.

Reply:
The multisig case doesn't seem to have the animation lockup issue, so I don't think we need this.

Reply:
I've tested it and still locks up though when searching for a lot of addresses, e.g.: from index 0 to 60 locks up a few times when generating the addresses to compare. Sleep fixes it.